evap/staff/tests/test_views.py

```diff
@@ -1733,6 +1754,15 @@ def test_copy_participants(self):
 
         self.assertEqual(self.evaluation.participants.count(), original_participant_count + self.evaluation2.participants.count())
 
+    def test_replace_copy_participants(self):
+        page = self.app.get(self.url, user=self.manager)
+
+        form = page.forms["participant-copy-form"]
+        form["evaluation"] = str(self.evaluation2.pk)
+        page = form.submit(name="operation", value="copy-participants")
+
+        self.assertEqual(self.evaluation.participants.count(), self.evaluation2.participants.count())
```

Comment:
If this test did the wrong thing and still passed, we should think about how to prevent this in the future.